plugins/Stk/Mallets/Mallets.cpp

```diff
@@ -469,7 +500,7 @@ QWidget * MalletsInstrumentView::setupModalBarControls( QWidget * _parent )
 	m_stickKnob->setLabel( tr( "Stick mix" ) );
 	m_stickKnob->move( 190, 90 );
 	m_stickKnob->setHintText( tr( "Stick mix:" ), "" );
-	
+
```

Comment:
```suggestion
```
Redundant I presume?

Reply:
I think you've only looked at the diff and not the whole file. The definition of the knobs follow a certain pattern and this hasn't changed. I just removed a trailing tab. A no on this one... ;)

Reply:
> I think you've only looked at the diff and not the whole file. The definition of the knobs follow a certain pattern and this hasn't changed. I just removed a trailing tab. A no on this one... ;)

Ah, alright.